frontend/app/src/pages/superadmin/header/HeaderStyles.tsx

```diff
@@ -19,6 +19,11 @@ export const AlternateWrapper = styled.div`
   justify-content: space-between;
   align-items: center;
   flex-shrink: 0;
+  position: fixed;
+  top: 62px;
+  left: 0;
+  width: 100%;
+  z-index: 9999999;
```

Comment:
this `z-index` doesnt feel like best practice is there a style to keep it always ontop? if you can look into alternatives to doing this that would be good

Reply:
I'm just going to make this another issue so we can merge this PR for now